### PR TITLE
add hw5

### PR DESCRIPTION
**Pull request by yanaPy**

### CODE REVIEW

File: hw5/FindMaxMinValue.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package hw4;
+
+public class FindMaxMinValue {
+
+    //2. Написати програму, яка знаходить мінімальне і максимальне число в масиві.
+    static void findMaxMinValue(int[] array) {
```
Comment (AndrewSkoba) [inline hw5/FindMaxMinValue.java]: Looks like you do not check the last element in array

File: hw5/ChangeArray.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package hw4;
+
+import java.util.Arrays;
+
+public class ChangeArray {
```
Comment (AndrewSkoba) [inline hw5/ChangeArray.java]: Looks like you do not check the last element in array. So, your last element always will has default value. In such a case 0.

File: hw5/PrintNumberNegativeAndSumOddElements.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+    /*   5. Написати програму яка виводить кількість відємних елементів нище головної діагоналі та
+    суму не парних елементів вище головної діагоналі двомірного масиву.*/
+    static void printNumberNegativeAndSumOddElements(int[][] array){
+
+        int numberOfNegative= 0;
+        int sumOfOddElements = 0;
+
+
+        for(int i=0; i< array.length;i++){
```
Comment (AndrewSkoba) [inline hw5/PrintNumberNegativeAndSumOddElements.java]: Pls, add space between operators

File: hw5/TableOfValues.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+/* 4. Написати програму, яка друкує таблицю значень sin(x),
+        де х - від 0 до 360 з кроком 10.(значення мають бути записані в масив за допомогою циклу)*/
+public class TableOfValues {
+
+    static void tableOfValues() {
+        double[] array = new double[37];
+        int j = 0;
+
+        for (int i = 0; i < array.length; i++) {
+            array[i] =  Math.sin(i);
```
Comment (AndrewSkoba) [inline hw5/TableOfValues.java]: So, in our array will be next values sin(0), sin(1), sin(2)..... sin(36)
Pls, fill array like sin(0), sin(10), sin(20)...sin(360)

File: hw5/PrintNumberNegativeAndSumOddElements.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+            for (int j = 0; j < array[0].length; j++) {
+                if (i < j&&array[i][j]% 2 == 0) {
+                    sumOfOddElements += (array[i][j]);
```
Comment (AndrewSkoba) [inline hw5/PrintNumberNegativeAndSumOddElements.java]: brackets redundant

File: hw5/FindMaxMinValue.java (hunk 1 above)
Comment (yanaPy) [reply hw5/FindMaxMinValue.java]: initialized var max and min as first var from array and cycle started with "int i = 1"

File: hw5/ChangeArray.java (hunk 2 above)
Comment (yanaPy) [reply hw5/ChangeArray.java]: deleted -1 from "i < array.length-1"

File: hw5/PrintNumberNegativeAndSumOddElements.java (hunk 3 above)
Comment (yanaPy) [reply hw5/PrintNumberNegativeAndSumOddElements.java]: done

File: hw5/TableOfValues.java (hunk 4 above)
Comment (yanaPy) [reply hw5/TableOfValues.java]: changed Math.sin(i) to Math.sin(j)

File: hw5/FindMaxMinValue.java (hunk 1 above)
Comment (AndrewSkoba) [reply hw5/FindMaxMinValue.java]: I have no idea why this comment here) It was for some other person.
I am sorry!

File: hw5/PrintNumberNegativeAndSumOddElements.java (hunk 5 above)
Comment (yanaPy) [reply hw5/PrintNumberNegativeAndSumOddElements.java]: done

**Review by AndrewSkoba — APPROVED** [review]

**Pull request merged**